### ISSUE TITLE
hyperparameters for model training

### ISSUE THREAD

**Issue by woshixyr** [LianpoWang/MaskDICNet#16]
The code shows the implementation of the inference part. We would like to know the key hyperparameters used during model training (such as learning rate, number of epochs, batch size, etc.).
